Set production options for css-loader

ikalnytskyi opened the pull request:
There are two things in this patch. First, we start producing CSS source
maps and this is very important, because we use CSS modules which mangle
CSS rules, i.e. change their names making unreal to map CSS classed back
to sources. Second, we minimize produced CSS for production builds
decreasing size-to-fetch for browsers.

ikalnitsky commented:
@olha-kurkaiedova thank you very much for your effort! unfortunately, i have bad news for you: we are going to use [Leiningen](http://leiningen.org) tool which is widely used in Clojure community. So I it seems we don't need gulpfile anymore.

ikalnytskyi committed:
Set production options for css-loader

There are two things in this patch. First, we start producing CSS source
maps and this is very important, because we use CSS modules which mangle
CSS rules, i.e. change their names making unreal to map CSS classed back
to sources. Second, we minimize produced CSS for production builds
decreasing size-to-fetch for browsers.
